modules/ROOT/pages/event-tracking.adoc

```diff
@@ -186,29 +194,33 @@ services:
 ----
 
 
+=== Complex Tracking Example
+
+This example sets tracking levels and settings at all three levels: global, Mule app, and flow.
```

Comment:
ditto confusion about two types of "levels"
Also, we have two uses of "set" and "settings" as well. Yikes.